Makefile

```diff
@@ -9,14 +9,14 @@ all: list
 
 install:
 	./tools/generate_key_pair.sh
-	pip install --upgrade -r requirements.dev.txt
+	pip install --upgrade -e .[develop]
 	export PRIVATE_KEY=`cat private.pem`
```

Comment:
@Mikanebu that's because `export PRIVATE_KEY=cat private.pem`is not valid bash command. 
should be `export PRIVATE_KEY=$(cat private.pem)`